scripts/pre_push_hook.py

```diff
@@ -0,0 +1,184 @@
+#!/usr/bin/env python2.7
+"""Pre-push hook that executes the Python/JS linters on all files that
+deviate from develop.
+(By providing the list of files to `scripts/pre_commit_linter.py`)
+To install the hook manually simply execute this script with the `--install`
+flag.
+"""
+
+import os
+import sys
+import subprocess
+import collections
+import pprint
+import argparse
+import shutil
+
+_PARSER = argparse.ArgumentParser()
+_PARSER.add_argument("remote", nargs="?", help="provided by git before push")
+_PARSER.add_argument("url", nargs="?", help="provided by git before push")
+_PARSER.add_argument("--install", action="store_true", default=False,
+                     help="Install pre_push_hook to the .git/hooks directory")
+_ARGS = _PARSER.parse_args()
```

Comment:
This is probably better placed in main because that's closer to where we are using it.